ast2vec/tests/test_repo2source.py

```diff
@@ -0,0 +1,41 @@
+import os
+import unittest
+
+import asdf
+
+import ast2vec.tests as tests
+import ast2vec.tests.models as paths
+from ast2vec import Source, Repo2SourceTransformer
+from ast2vec.enry import install_enry
+from ast2vec.tests.models import DATA_DIR_SOURCE
+
+
+def validate_asdf_file(obj, filename):
+    data = asdf.open(filename)
+    obj.assertIn("meta", data.tree)
+    obj.assertIn("sources", data.tree)
+    obj.assertIn("uasts", data.tree)
+    obj.assertEqual(0, len(data.tree["meta"]["dependencies"]))
```

Comment:
Add better assertions. Try to deserialize UAST. Try to find substrings in some file with a predefined name. Check meta["model"] to be "source"